src/brasil/gov/portal/upgrades/v10700/handler.py

```diff
@@ -30,7 +32,16 @@ def atualiza_produtos_terceiros(context):
         'collective.polls:default',
         'sc.embedder:default',
         'sc.social.like:default',
-    )
+    ]
+
+    # Somente executo o upgrade step do brasil.gov.agenda se o seu profile foi
+    # instalado. Em versões antigas do brasil.gov.portal o brasil.gov.agenda
+    # não era instalado.
+    ps = api.portal.get_tool('portal_setup')
```

Comment:
the context is the `portal_setup` tool; there's no need to get it, just change the parameter name to increase readability.